metricbeat/module/kibana/testing.go

```diff
@@ -0,0 +1,32 @@
+package kibana
```

Comment:
Move common testing code to `kibana/mtest` as discussed in https://github.com/elastic/beats/pull/7106#discussion_r188708832?

Reply:
done